src/dsp_tools/xmllib/models/resource.py

```diff
@@ -884,6 +888,11 @@ def add_richtext_multiple(
         Returns:
             The original resource, with the added values
         """
+        # Because of the conversion, the richtext is converted to a string values such as `pd.NA`
+        # are not recognised as empty any more.
+        # Therefore, this additional check is required here.
```

Comment:
I don't understand this comment. You can merge now, but perhaps afterwards we can discuss what this should mean, and if we want to improve its wording.